plugins/embedding.cc

```diff
@@ -1334,6 +1336,269 @@ regNearestNeighborsFunction(builtinPackage(),
                             "Return the nearest neighbors of a known row in an embedding dataset",
                             "functions/NearestNeighborsFunction.md.html");
 
+/*****************************************************************************/
+/* Image Wrapper function                                                    */
+/*****************************************************************************/
+
+DEFINE_STRUCTURE_DESCRIPTION(ImageWrapperFunctionConfig);
+
+ImageWrapperFunctionConfigDescription::
+ImageWrapperFunctionConfigDescription()
+{
+    addField("expression",
+             &ImageWrapperFunctionConfig::expression,
```

Comment:
We do need to document the parameters here.  We can then auto-generate the documentation and it's sure to stay up to date with the code.